docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -300,6 +300,9 @@ merged through the normal process.
 
 In [Appendix B] it is shown how `merge-develop-into-master` can be verified.
 
+Create a pull request from `master` on github back into `develop` and have it
+merged through the normal process.
+
```

Comment:
It is a long section, and the second merge can easily missed. Secondly, it should be clarified that you must wait for the first merge before you can create the second.

Please update with
```
### 16.2 Merge master into develop

When the previous pull request has been merged, create a pull request from `master` on github back
into `develop` and have it merged through the normal process.
```

On line 282 add
```
### 16.1 Merge develop into master
```